tools/generator/GenBase.cs

```diff
@@ -530,13 +530,21 @@ bool CanMethodBeIsStyleSetter (Method m)
 		}
 
 		// Keep sorted alphabetically
-		static readonly string[]    ObjectRequiresNew       = new string[]{
-			"Handle",
-		};
-
-		static readonly string[]    ThrowableRequiresNew    = new string []{
-			"Message",
-		};
+		static readonly string[]    ObjectRequiresNew       = typeof (object)
+			.GetMethods ()
+			.Select (m => m.Name)
+			.Concat (new[] { "Handle" })
+			.OrderBy (v => v, StringComparer.OrdinalIgnoreCase)
+			.ToArray ();
+
+
+		static readonly string[]    ThrowableRequiresNew    = typeof (System.Exception)
+			.GetMethods ()
+			.Select (m => m.Name)
+			.Concat (typeof (System.Exception).GetProperties ().Select (p => p.Name))
+			.Concat (new[] { "Handle" })
+			.OrderBy (v => v, StringComparer.OrdinalIgnoreCase)
+			.ToArray ();
```

Comment:
Hmm, this does seem like it would be correct, but how many methods are actually here?

It seems like it might be better to just list them all and not use reflection? New methods/properties on `System.Object` and `System.Exception` aren't going to appear later on, right?